src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -76,6 +76,48 @@ public void testListUserStatusTransitions() throws Exception {
         assertThat(returnValue.getData().get(0).getToStatus(), is(equalTo(DE_ACTIVATED)));
     }
 
+    @Test
+    public void testListUsers_withNoParameters() throws Exception {
+        String functionality = "listUsers";
+        initMockServer(functionality);
+
+        HyperwalletList<HyperwalletUser> returnValue;
+        try {
+            returnValue = client.listUsers();
+        } catch (Exception e) {
+            mockServer.verify(parseRequest(functionality));
+            throw e;
+        }
+        assertThat(returnValue.getCount(), is(equalTo(2)));
```

Comment:
Pls add assert statement for the 2nd index as well.

Reply:
Done